### PR TITLE
Add some player count conditions to map rotation

### PR DESCRIPTION
**Pull request by necessarily-equal**
I've set fairly conservative values, IMO

### CODE REVIEW

File: dist/configs/game/maprotation.cfg [hunk 1]
```diff
@@ -1,16 +1,22 @@
 rotation1
 {
 	antares
-	chasm
+	if numPlayers > 6 #skip-chasm
```
Comment (slipher) [inline dist/configs/game/maprotation.cfg:4]: I kind of see 6 as the minimum number of players for the game to be any good... so it is a bit strange to skip for as few as 7 players. 

Also there might be some players idle/spectating.

Comment (necessarily-equal) [reply dist/configs/game/maprotation.cfg]: Ok, I've bumped it to some larger value, to skip if `>9` players

File: dist/configs/game/maprotation.cfg [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+	if numPlayers > 9 #skip-chasm
+		chasm
+	#skip-chasm
 	forlorn
 	parpax
 	perseus
 	plat23
-	spacetracks
+	if numPlayers < 6 #skip-spacetracks
+		spacetrack
```
Comment (slipher) [inline dist/configs/game/maprotation.cfg:12]: Typos on this line and the next

**Comment by illwieckz** [comment]
We now have a new syntax without having to use such goto/label mechanism.

Now we can just do:

```
if numPlayers > [number] [mapname]
```

And if `numPlayers` is not satisfied, the game will just skip to the next line.

For example, this is my rotation on my server:

```
rotation1 {
 chasm
 usstremor
 yocto
 parpax
 arachnid2
 ctcs
 vega
 atcshd
 if numPlayers > 6 antares
 if numPlayers > 11 hangar28
 if numPlayers > 5 area3
 if numPlayers > 3 station15
 plat23
 if numPlayers > 7 transit
 perseus
 karith
 spacetracks
 if numPlayers > 11 thunder
 if numPlayers > 5 nexus6
 if numPlayers > 7 jota
 forlorn
 parpax
 usstremor
 karith
 yocto
 perseus
 ctcs
 antares
 arachnid2
 plat23
 uncreation
 if numPlayers > 5 vega
 if numPlayers > 3 spacetracks
 if numPlayers > 5 transit
 forlorn // first loaded by server.cfg
}
```

**Comment by illwieckz** [comment]
So if I'm right, your proposed rotation can be translated to:

```c
rotation1
{
	antares
	if numPlayers < 10 chasm
	forlorn
	parpax
	perseus
	plat23
	if numPlayers > 5 spacetracks
	station15
	if numPlayers > 9 thunder
	if numPlayers > 3 vega
	yocto
}
```

File: dist/configs/game/maprotation.cfg (hunk 2 above)
Comment (necessarily-equal) [reply dist/configs/game/maprotation.cfg]: copy-paste error :X

**Comment by necessarily-equal** [comment]
I've updated it with the shorter syntax

**Comment by illwieckz** [comment]
```
2021-03-28 22:42:07 +0200 <Unvanquished_>	13 people (6+4 A, 7+3 H, 0 S) playing chasm on [DoH] Experimental - unv://109.230.195.81:27961
```

This was fun. Maybe raise the chasm limit to `< 15` ?

Note: `< 15` because 7v7 (14) would fit.

**Comment by necessarily-equal** [comment]
I think this is ready

**Pull request merged**